api-client/src/endpoints/createLocalNewsEndpoint.ts

```diff
@@ -19,16 +18,14 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<LocalNewsModel>> =>
     )
     .withMapper(
       (json: Array<JsonLocalNewsType>): Array<LocalNewsModel> =>
-        json.map((localNews: JsonLocalNewsType) => {
-          const decodedTitle = parseHTMLEntities(localNews.title)
-          const decodedMessage = parseHTMLEntities(localNews.message)
-
-          return new LocalNewsModel({
-            id: localNews.id,
-            timestamp: moment.tz(localNews.timestamp, 'GMT'),
-            title: decodedTitle,
-            message: decodedMessage
-          })
-        })
+        json.map(
+          (localNews: JsonLocalNewsType) =>
+            new LocalNewsModel({
+              id: localNews.id,
+              timestamp: moment.tz(localNews.timestamp, 'GMT'),
```

Comment:
since 'GMT' is used quite offen in several files i would prefer using a const here

Reply:
Since I did not change anything here and moment will be removed anyway (https://issues.tuerantuer.org/browse/IGAPP-281) I will keep it as it is.